install-ci.yml

```diff
@@ -247,3 +247,12 @@
     - role: dd-apache
       tags:
         - monitoring
+
+# This task needs to be done last because fail2ban will fail to start
+# if the configured log files to scan do not exist yet.
```

Comment:
Yes, we could touch the file first, but not necessarily in the fail2ban role because not all hosts enable the same jails, and the needed logpath is defined per jail.